core/src/main/resources/hudson/model/Messages_fr.properties

```diff
@@ -158,14 +158,14 @@ RunParameterDefinition.DisplayName=Param\u00e8tre d''ex\u00e9cution
 PasswordParameterDefinition.DisplayName=Param\u00e8tre "Mot de passe"
 
 Node.Mode.NORMAL=Utiliser ce noeud autant que possible
-Node.Mode.EXCLUSIVE=R\u00e9server cette machine pour les jobs qui lui sont attach\u00e9s seulement
+Node.Mode.EXCLUSIVE=R\u00e9server ce noeud uniquement pour les projets qui lui sont attach\u00e9s
 
 ListView.DisplayName=Vue liste
 
 MyView.DisplayName=Ma vue
 
-LoadStatistics.Legends.TotalExecutors=Nb total d''ex\u00e9cuteurs
-LoadStatistics.Legends.BusyExecutors=Nb d''ex\u00e9cuteurs occup\u00e9s
+LoadStatistics.Legends.TotalExecutors=Nombre total d''ex\u00e9cuteurs
```

Comment:
Note that the columns are supposed to be short. Better to double-check the UI

Reply:
<details>

<summary>Screenshot</summary>

### Load statistics

![Screenshot-2020-10-18_11-58-15](https://user-images.githubusercontent.com/2662497/96364125-47c78c00-1139-11eb-83b5-6a93f02d5bdf.png)


</details>

AFAICT it's not this one that is used as a column label.